imageroot/actions/configure-module/20config

```diff
@@ -185,6 +185,39 @@ if "timezone" in data and data["timezone"] != os.environ["WEBTOP_TIMEZONE"]:
     agent.set_env("RESTART_WEBDAV", "1")
     agent.set_env("RESTART_Z_PUSH", "1")
 
+# adjust the ejabberd module, when we disabled globally the chat module we use '-' as ejabberd module (needed by NSCombobox)
+if "ejabberd_module" in data and data["ejabberd_module"] != '-' and "ejabberd_domain" in data:
```

Comment:
This is incorrect, if you put `-` in the `ejabberd_module` you simply skip all this part, leaving the current configuration untouched.

* If you want di disable the configuration just set `ejabberd_module` as an empty string, as stated in row 195.
* Please don't mix frontend logic with backend logic.
```suggestion
# adjust the ejabberd module, when we disabled globally the chat module we use "" as ejabberd module
if "ejabberd_module" in data and "ejabberd_domain" in data:
```

Reply:
UI needs tricks also to be workable as backend :/

Reply:
it should be good @Amygos